### PR TITLE
EPMEDUTOUR-11625 - validation fixes, EPMEDUTOUR-11626 - social tags

### PR DESCRIPTION
**Pull request by error404as**
### What does this PR do:
Added meta tags for Facebook/Twitter.
Fixed html validator errors (not all, the only left - not allowed html tags inside <button>, error looks not correct)

#### Ticket Links:
https://jira.epam.com/jira/browse/EPMEDUTOUR-11625 - Fix validation errors related to HTML structure
https://jira.epam.com/jira/browse/EPMEDUTOUR-11626 - Add social Media Meta Tags

### CODE REVIEW

**Review by ashumski-adnoc — APPROVED** [review]

File: src/_components/sectionFeedbacks.ejs [hunk 1]
```diff
@@ -13,7 +13,7 @@ if (!sectionFeedbacks[locale]) { locale = site.defaultLocale; }
                 <div class="swiper-slide c-feedbacks__item">
                     <div class="c-feedbacks__item-head">
                         <div class="c-feedbacks__item-userpic">
-                            <img src="https://eu.ui-avatars.com/api/?background=random&name=<%= post.author %>&size=48" alt="<%= post.author %>" aria-hidden="true">
+                            <img src="https://eu.ui-avatars.com/api/?background=random&name=<%= post.author.replace(/\s/g, '%20') %>&size=48" alt="<%= post.author %>" aria-hidden="true">
```
Comment (alexeykomov) [inline src/_components/sectionFeedbacks.ejs:16]: Maybe we need to do full URL encoding here?
```js
encodeURIComponent('Alexey Komov with &');
"Alexey%20Komov%20with%20%26"
```

File: src/_layouts/default.ejs [hunk 2]
```diff
@@ -20,23 +20,39 @@ var pageDescription = locals.description || site[locale].description;
...
+<!-- Twitter -->
+<meta name="twitter:card" content="summary">
+<meta name="twitter:url" content="<%= site.url %>" />
+<meta name="twitter:title" content="<%= locals.title %>">
+<meta name="twitter:description" content="<%= pageDescription %>">
+<meta name="twitter:image" content="/images/og-1200x628.png">
+
 <!-- Yandex.Metrika counter -->
-<script type="text/javascript" >
+<script>
```
Comment (alexeykomov) [inline src/_layouts/default.ejs:40]: 👍

Comment (alexeykomov) [reply src/_layouts/default.ejs:40]: Can it be `async` or `defer`?

File: src/_components/sectionFeedbacks.ejs (hunk 1 above)
Comment (error404as) [reply src/_components/sectionFeedbacks.ejs:16]: Ru names become too long. It works as now, so let's keep it like this

File: src/_layouts/default.ejs (hunk 2 above)
Comment (error404as) [reply src/_layouts/default.ejs:40]: No recommended, as I remember

**Review by alexeykomov — APPROVED** [review]

**Pull request merged**